tests/lib/rules/jsx-handler-names.js

```diff
@@ -265,7 +265,7 @@ ruleTester.run('jsx-handler-names', rule, {
       ],
     },
     {
-      code: '<TestComponent only={this.handleChange} />',
+      code: '<TestComponent2 only={this.handleChange} />',
```

Comment:
This looked like an intent to test both `message`, and `messageId` + `data` with two tests in a row, so I just modified one of those two a bit.